test/test_torch.py

```diff
@@ -1653,14 +1655,6 @@ def reference(x, k, o3, o32):
         self._test_conv_corr_eq(lambda x, k: torch.xcorr3(x, k), reference)
 
     @unittest.skip("Not implemented yet")
-    def test_xcorr3_xcorr2_eq(self):
```

Comment:
the one with the 'F' seems to be the full correlation, and the other `valid`. Maybe add a `_full` somewhere? :)